config/application.config.php

```diff
@@ -8,6 +8,8 @@
 return array(
     // This should be an array of module namespaces used in the application.
     'modules' => array(
+        'Zend\Router',
```

Comment:
Not currently. The `ModuleResolverListener` always appends `Module` to the module name in order to determine if the module exists.